fpga_interchange/arch.cc

```diff
@@ -1767,6 +1767,9 @@ bool Arch::checkPipAvailForNet(PipId pip, NetInfo *net) const
         NPNR_ASSERT(src != wire);
         NPNR_ASSERT(dst != wire);
 
+        if (tile_type.lut_elements.size() > 0)
```

Comment:
looks like this is disabling all pseudo-pips in tiles with LUTs, are you sure you want to do this?

Reply:
I;m testing it right now, if it's really required. I had issues before, but maybe they are fixed now.

Reply:
So with out this lines Nextpnr doesn't work for xilinx 7-series, as it uses pseudo-PIPs going through (A|B|C|D)6LUTs, but it doesn't set A6 pin high making this pseudo-PIP broken

Reply:
So I've disabled offending pseudoPIPs in python-fpga-interchange. This code is no longer used